<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11725" started-at="2014-09-14T08:02:09Z" finished-at="2014-09-14T08:02:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11725" started-at="2014-09-14T08:02:09Z" finished-at="2014-09-14T08:02:21Z">
      <class name="com.example.tests.ContactCreationTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.ContactCreationTest@4c64881d]" name="setUp" is-config="true" duration-ms="7161" started-at="2014-09-14T11:02:09Z" finished-at="2014-09-14T11:02:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTest@4c64881d]" name="testNonEmptyContactCreation" duration-ms="2540" started-at="2014-09-14T11:02:17Z" data-provider="contactsFromFile" finished-at="2014-09-14T11:02:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [firstName=testName-662863044yeekpxgexyrjrzxbqayy, lastName=testName-1734990798vgdukqikozqlevtgkkki]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-662863044yeekpxgexyrjrzxbqayy, lastName=testName-1734990798vgdukqikozqlevtgkkki], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]><![CDATA[]]]]><![CDATA[>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-662863044yeekpxgexyrjrzxbqayy, lastName=testName-1734990798vgdukqikozqlevtgkkki], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at com.example.tests.ContactCreationTest.testNonEmptyContactCreation(ContactCreationTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation(com.example.tests.ContactData)[pri:0, instance:com.example.tests.ContactCreationTest@4c64881d]" name="testNonEmptyContactCreation" duration-ms="1843" started-at="2014-09-14T11:02:19Z" data-provider="contactsFromFile" finished-at="2014-09-14T11:02:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[ContactData [firstName=, lastName=testName-1483543042amurymxynwnudcupcdgc]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName-1483543042amurymxynwnudcupcdgc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1483543042amurymxynwnudcupcdgc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]><![CDATA[]]]]><![CDATA[>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName-1483543042amurymxynwnudcupcdgc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
     but: was <[ContactData [firstName=, lastName=testName-1078310379dkgtjybazfjwokhktwyc], ContactData [firstName=, lastName=testName1904302567lcxrcckkyzigikuemqix], ContactData [firstName=testName-1002704647ebuuaiwmrcklhnhyconc, lastName=], ContactData [firstName=testName-1483543042amurymxynwnudcupcdgc, lastName=], ContactData [firstName=testName-1734990798vgdukqikozqlevtgkkki, lastName=testName-662863044yeekpxgexyrjrzxbqayy], ContactData [firstName=testName1133716067syjvpmboptmgxnfamfkw, lastName=testName1513875977fhsglxxiwlnvcrrnwpwd]]]]><![CDATA[>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert.assertThat(Assert.java:832)
	at com.example.tests.ContactCreationTest.testNonEmptyContactCreation(ContactCreationTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactCreationTest@4c64881d]" name="tearDown" is-config="true" duration-ms="159" started-at="2014-09-14T11:02:21Z" finished-at="2014-09-14T11:02:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.ContactCreationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
